templates/csi-clusterrole.yaml

```
@@ -1,3 +1,6 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
 {{- template "vault.csiEnabled" . -}}
 {{- if .csiEnabled -}}
 apiVersion: rbac.authorization.k8s.io/v1
```

I think the CI failures are being caused by the formerly empty templates appearing with just the copyright header, which helm/kubectl err's on.

```suggestion
{{- template "vault.csiEnabled" . -}}
{{- if .csiEnabled -}}
# Copyright (c) HashiCorp, Inc.
# SPDX-License-Identifier: MPL-2.0

apiVersion: rbac.authorization.k8s.io/v1
```

---

Oh, actually I think we'll want template comments, instead of yaml comments, so that the copyright isn't rendered by helm: https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments

So it would look something like this:

```suggestion
{{/*
Copyright (c) HashiCorp, Inc.
SPDX-License-Identifier: MPL-2.0
*/}}

{{- template "vault.csiEnabled" . -}}
{{- if .csiEnabled -}}
apiVersion: rbac.authorization.k8s.io/v1
```